_maps/RandomRuins/IceRuins/icemoon_surface_phonebooth.dmm

```diff
@@ -0,0 +1,108 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"a" = (
+/obj/machinery/holopad,
+/turf/open/floor/iron/dark,
+/area/ruin/unpowered/Phone_booth)
+"c" = (
+/obj/machinery/door/window{
+	dir = 1
+	},
+/obj/machinery/door/window,
+/turf/open/floor/iron/dark/smooth_edge,
+/area/ruin/unpowered/Phone_booth)
+"e" = (
+/turf/open/space/basic,
```

Comment:
You have space turfs on a planetary ruin. These should be replaced with an appropriate planetary turf.